### PR TITLE
Bump to lotus 1.15.1-rc5

### PR DESCRIPTION
**Pull request by ribasushi**
Depends on https://github.com/application-research/filclient/pull/65 ( already pulls it into the go.mod )

Also resolves https://github.com/ipfs/go-path/issues/56 for @gmelodie 

NOTE: UNTESTED!!! ( because we don't have tests yet ;)

### CODE REVIEW

**Review by gmelodie — APPROVED** [review]
Looks good! Thanks for doing this!

**Comment by ribasushi** [comment]
@gmelodie you are too quick
The `go.mod` you merged refers to an **unmerged** commit from `filclient`. See the `Depends on...` 1st line above.

File: go.mod [hunk 1]
```diff
@@ -86,6 +86,298 @@ require (
 	gorm.io/gorm v1.21.15
 )
 
+require (
```
Comment (en0ma) [inline go.mod:89]: @ribasushi is this second `require` not weird?

Comment (ribasushi) [reply go.mod:89]: @en0ma no, it is a go1.17+ thing. Note how everything is `indirect`

**Pull request merged**